presidio-anonymizer/presidio_anonymizer/entities/analyzer_results.py

```diff
@@ -0,0 +1,63 @@
+"""Wrapper to the list of AnalyzerResult which manipulates the list."""
+from typing import List
+
+
+class AnalyzerResults(List):
+    """
+    Receives the analyzer result list and manipulates it.
+
+     The manipulation contains removal of unused results and sort by indices order.
+     According to the logic of:
+    - One PII - uses a given or default transformation to anonymize and replace the PII
+    text entity.
+    - Full overlap of PIIs - When one text have several PIIs, the PII with the higher
+    score will be taken.
+    Between PIIs with similar scores, the selection will be arbitrary.
+    - One PII is contained in another - anonymizer will use the PII with larger text.
+    - Partial intersection - both will be returned concatenated.
+    """
+
+    def to_sorted_set(self, reverse=False):
+        """
+        Manipulate the list.
+
+        remove_dups - removes results which impact the same text and should be ignored.
```

Comment:
I don't understand where all the flags are used?
For example, where to put `remove_dups`? is it an input somewhere? 
I was looking for it in the code and I don't understand it in the context of this function which has 2 lines of code.
As a contributor in the future of presidio, what should I do with this info? Is it the correct place for it? should be move to a different function?

Reply:
DONE